test/libsolidity/semanticTests/inlineAssembly/blobhash_empty.sol

```diff
@@ -0,0 +1,11 @@
+contract C {
+    function f() public view returns (bytes32 ret) {
+        assembly {
+            ret := blobhash(3)
```

Comment:
I'd add a comment that this being empty depends on the fact that we have 2 blobs hard-coded for the current block in `EVMHost`. Out of that context is not at all clear why you'd be guaranteed a 0 here.

Also changing the name to something like `blobhash_index_exceeding_blob_count.sol` would also add more clues.